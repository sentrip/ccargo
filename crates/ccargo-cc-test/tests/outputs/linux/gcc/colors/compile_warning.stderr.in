[01m[Kmain.c:[m[K In function â€˜[01m[Kmain[m[Kâ€™:
[01m[Kmain.c:4:14:[m[K [01;35m[Kwarning: [m[Kinitialization of â€˜[01m[Kint *[m[Kâ€™ from incompatible pointer type â€˜[01m[Klong long int *[m[Kâ€™ [[01;35m[K-Wincompatible-pointer-types[m[K]
    4 |     int* x = [01;35m[K&[m[Kv;
      |              [01;35m[K^[m[K
[01m[Kmain.c:5:14:[m[K [01;35m[Kwarning: [m[Kinitialization of â€˜[01m[Kint *[m[Kâ€™ from incompatible pointer type â€˜[01m[Klong long int *[m[Kâ€™ [[01;35m[K-Wincompatible-pointer-types[m[K]
    5 |     int* y = [01;35m[K&[m[Kv;
      |              [01;35m[K^[m[K
