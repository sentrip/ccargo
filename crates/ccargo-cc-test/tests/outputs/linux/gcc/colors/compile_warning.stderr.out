[0m[1m[33mwarning[0m[1m: [0m[m[Kinitialization of â€˜[01m[Kint *[m[Kâ€™ from incompatible pointer type â€˜[01m[Klong long int *[m[Kâ€™ [[01;33m[K-Wincompatible-pointer-types[m[K][0m[1m[36m
   --> [0mmain.c:4:14 in function `main`[0m
    4 |     int* x = [01;33m[K&[m[Kv;[0m
      |              [01;33m[K^[m[K[0m
[0m[1m[33mwarning[0m[1m: [0m[m[Kinitialization of â€˜[01m[Kint *[m[Kâ€™ from incompatible pointer type â€˜[01m[Klong long int *[m[Kâ€™ [[01;33m[K-Wincompatible-pointer-types[m[K][0m[1m[36m
   --> [0mmain.c:5:14 in function `main`[0m
    5 |     int* y = [01;33m[K&[m[Kv;[0m
      |              [01;33m[K^[m[K[0m
